src/main/java/haveric/recipeManager/recipes/CraftRecipeParser.java

```diff
@@ -88,7 +88,7 @@ public boolean parseRecipe(int directiveLine) throws Exception {
 
         recipe.setIngredients(ingredients); // done with ingredients, set 'em
 
-        if (recipe.hasFlag(FlagType.REMOVE)) {
+        if (recipe.hasFlag(FlagType.REMOVE) && !Version.has1_12Support()) {

```

Comment:
If I replace the ones in Remove with the matches from a fresh Extract, the errors do go away.